pkg/detectors/brandfetch/brandfetch.go

```diff
@@ -47,15 +48,13 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		}
 
 		if verify {
-			payload := strings.NewReader(`{
-				"domain": "www.example.com"
-				}`)
-			req, err := http.NewRequestWithContext(ctx, "POST", "https://api.brandfetch.io/v1/color", payload)
+			// API upgraded to v2 from v1, new API doc:  https://docs.brandfetch.com/reference/brand-api
+			req, err := http.NewRequestWithContext(ctx, "GET", "https://api.brandfetch.io/v2/brands/google.com", nil)
```

Comment:
The current `v1` APIs with the old tokens are still functioning properly. Therefore, rather than updating the existing version, I would recommend adding a new version of the detector.